apps/src/codebridge/Settings/SettingsDropdown.tsx

```diff
@@ -83,10 +82,25 @@ const SettingsDropdown: React.FunctionComponent<SettingsDropdownProps> = ({
   const dropdownStyles = useDropdownPosition(buttonRef, dropdownRef);
 
   const onTextEditorDropdownChange = (value: string) => {
-    setSelectedEditorFontSizeValue(getSelectedKey(value));
+    const selectedEditorKey = getSelectedKey(value);
+    setSelectedEditorFontSizeValue(selectedEditorKey);
+    handleFontSizeChange(
+      'CodeEditor',
+      selectedEditorKey,
+      currentEditorFontSizeKey,
+      EVENTS.CODEBRIDGE_EDITOR_FONT_SIZE_CHANGE
+    );
   };
+
   const onConsoleDropdownChange = (value: string) => {
+    const selectedConsoleKey = getSelectedKey(value);
     setSelectedConsoleFontSizeValue(getSelectedKey(value));
```

Comment:
nit: use `selectedConsoleKey` here instead of calling the function again

Reply:
Oh yeah! Meant to do that. Thanks!